pkg/cmd/server/origin/run_components.go

```diff
@@ -243,13 +248,16 @@ func (c *MasterConfig) RunBuildController(informers shared.InformerFactory) erro
 	groupVersion := schema.GroupVersion{Group: "", Version: storageVersion}
 	codec := kapi.Codecs.LegacyCodec(groupVersion)
 
-	admissionControl, err := admission.InitPlugin("SecurityContextConstraint", c.KubeClientset(), "")
+	pluginInitializer := kubeadmission.NewPluginInitializer(
+		c.KubeClientsetInternal(),
+		c.Informers.InternalKubernetesInformers(),
+		nil, // api authorizer, only used by PSP
+		nil, // cloud config
```

Comment:
Oh wait, this is just admission for the build controller, which appears to only use it for SCC. I don't think we need to change anything. Agree?